metron-streaming/Metron-DataLoads/src/test/java/org/apache/metron/dataloads/nonbulk/taxii/flatfile/SimplEnrichmentFlatFileLoaderTest.java

```diff
@@ -0,0 +1,147 @@
+/**
```

Comment:
Why is this file moved to org.apache.metron.dataloads.nonbulk.taxii from org.apache.metron.dataloads.nonbulk?